src/helpers/ScrollToTop/ScrollToTop.jsx

```diff
@@ -27,7 +27,10 @@ class ScrollToTop extends React.Component {
    * @memberof ScrollToTop
    */
   componentDidUpdate(prevProps) {
-    if (this.props.location?.pathname !== prevProps.location?.pathname) {
+    if (
+      !this.props.location?.pathname.hash &&
```

Comment:
Typo. This should read
!this.props.location?.hash &&

Reply:
I took care of this in https://github.com/plone/volto/pull/4287/files#diff-1712f4ab0dc421cacb610886108bb1e5cd7978212604901774cf341a5fcf9ed1R30

Reply:
Also, we need to account for "copy link to highlight" feature on browsers. Right now it causes problems.